lean4-infoview/src/infoview/index.css

```diff
@@ -31,7 +31,8 @@ html,body {
 }
 
 .pre-wrap {
-    white-space: 'pre-wrap';
+    white-space: pre-wrap;
+    word-wrap: break-word;
```

Comment:
Did you find a case that needed [break-work](https://www.w3schools.com/cssref/playdemo.asp?filename=playcss_word-wrap&preval=break-word). I don't recall seeing very long words...?

Reply:
Yes! I'm still WIP checking a best solution for the parenthesis case (break-word resolve the issue with the parenthesis but break "big words" as you said)